tool/teleport/common/teleport_test.go

```diff
@@ -111,6 +127,21 @@ func TestTeleportMain(t *testing.T) {
 		require.Equal(t, "10.5.5.5", conf.AdvertiseIP)
 		require.Equal(t, map[string]string{"a": "a1", "b": "b1"}, conf.SSH.Labels)
 	})
+
+	t.Run("Bootstrap", func(t *testing.T) {
+		_, cmd, conf := Run(Options{
+			Args:     []string{"start", "--bootstrap", bootstrapFile},
+			InitOnly: true,
+		})
+		require.Equal(t, "start", cmd)
+		require.Equal(t, len(bootstrapEntries), len(conf.Auth.Resources))
+		for i, entry := range bootstrapEntries {
+			t.Log(i)
```

Comment:
Suggestion: replace index logging with t.Run(entry.kind)? Makes failures more readable.

Alternatively we could add entry.kind (or entry.fileName) to the error messages below, if you think 2 layers of t.Run is too much.

```suggestion
```

Reply:
I didn't mean to leave that in 😬

I made it so that we log `entry.FileName` in those checks, so it's immediately obvious which entry is failing the check, regardless.